multiqc/modules/rna_seqc/rna_seqc.py

```diff
@@ -107,15 +108,66 @@ def rnaseqc_general_stats (self):
             'scale': 'YlGn',
             'modify': lambda x: float(x) * 100.0
         }
+        headers['Expression Profiling Efficiency'] = {
+            'title': '% Expression Efficiency',
+            'description': 'Expression Profiling Efficiency',
+            'max': 100,
+            'min': 0,
+            'suffix': '%',
+            'scale': 'RdBu',
+            'modify': lambda x: float(x) * 100.0
+        }
         headers['Genes Detected'] = {
             'title': '# Genes',
             'description': 'Number of genes detected',
             'min': 0,
             'scale': 'Bu',
             'format': '{:,.0f}'
         }
+
         self.general_stats_addcols(self.rna_seqc_metrics, headers)
 
+    def transcript_associated_plot (self):
+        """ Plot a bargraph showing the Transcripts-associated reads  """
+        # Plot bar graph of groups
+        keys = [ 'Exonic Rate', 'Intronic Rate', 'Intergenic Rate' ]
```

Comment:
Not impossible - just use an `OrderedDict` instead of a list and set `name` to whatever you like (eg. `Exonic`). See http://multiqc.info/docs/#bar-graphs